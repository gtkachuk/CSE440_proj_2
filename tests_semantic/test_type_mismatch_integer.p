program testTypeMismatchInteger;

class testTypeMismatchInteger

BEGIN
   VAR 
      compilerWorks : boolean;
      jobDone	    : integer;

FUNCTION testTypeMismatchInteger;
BEGIN
   compilerWorks := jobDone
END

END
.

