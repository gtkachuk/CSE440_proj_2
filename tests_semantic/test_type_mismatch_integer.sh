#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_type_mismatch_integer.p
