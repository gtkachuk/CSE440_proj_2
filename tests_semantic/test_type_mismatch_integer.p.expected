ERROR:13:Type mismatch between "boolean" and "integer"
Errors detected. Exiting.
